<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveDialog</class>
 <widget class="QDialog" name="SaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据保存设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/resource/Save.png</normaloff>:/image/resource/Save.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="DirEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SelectButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>使用默认路径</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="SaveFreq">
         <item>
          <property name="text">
           <string>1Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100Hz</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="PathSelectButton">
         <property name="text">
          <string>确认修改</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="ReplayEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ReplaySelectButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="ReplayFreq">
         <item>
          <property name="text">
           <string>1Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200Hz</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CleanTrialButton">
         <property name="text">
          <string>清除航迹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Pause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartReplayButton">
         <property name="text">
          <string>开始回放</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
